post_office/models.py

```diff
@@ -57,6 +58,9 @@ class Email(models.Model):
     last_updated = models.DateTimeField(db_index=True, auto_now=True)
     scheduled_time = models.DateTimeField(_('The scheduled sending time'),
                                           blank=True, null=True, db_index=True)
+    expires_at = models.DateTimeField(_('Expires timestamp for email'),
```

Comment:
`Email won't be sent after this timestamp`

Reply:
done